src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -8240,6 +8240,7 @@
           - ElectricalEnergyMeasurement
           - EnergyPreference
           - ScenesManagement
+          - WaterHeaterManagement
```

Comment:
Please let Apple manage this file, unless this line was approved by them

Reply:
I have undone this change